conanfile.py

```diff
@@ -1,14 +1,15 @@
-#!/usr/bin/env python
 # -*- coding: utf-8 -*-
 
 import os
 from conans import ConanFile, CMake, tools
 from conans.errors import ConanInvalidConfiguration
+import shutil
+import tempfile
 
 
 class WinflexbisonConan(ConanFile):
-    name = "winflexbison"
-    version = "2.5.16"
+    name = "winflexbison_installer"
```

Comment:
Winflexbison is already in conan-center. We can't easily rename it now. See https://github.com/bincrafters/community/issues/489

Reply:
This is a newer version, so I don't think existing packages will break.

Reply:
I will revert to winflexbison